Gemfile.jruby-1.9.lock

```diff
@@ -126,12 +538,260 @@ GEM
     tins (1.3.4)
     treetop (1.4.15)
       polyglot
+      polyglot
+      polyglot
+      polyglot
+      polyglot
+      polyglot
+      polyglot
+      polyglot
+      polyglot
+      polyglot
+      polyglot
+      polyglot
+      polyglot
+      polyglot
+      polyglot
+      polyglot
+      polyglot
+      polyglot (>= 0.3.1)
```

Comment:
but wait, it should use the bundler installed in vendor/bundler which is the last version... !?